### PR TITLE
cc 695 tf profile cidev

### PR DESCRIPTION
**Pull request by mattch1**
JIRA: https://companieshouse.atlassian.net/browse/CC-695

Added cidev terraform profile values.

### CODE REVIEW

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 1]
```diff
@@ -0,0 +1,15 @@
+aws_bucket = "development-eu-west-2.terraform-state.ch.gov.uk"
```
Comment (mwestacott) [inline terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:1]: Need to check with @tobafolaranmi-ch on some of these. We only had `environment` and `aws_profile` defined for OE web.

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+remote_state_bucket = "ch-development-terraform-state-london"
+environment = "cidev"
+deploy_to = "development"
+state_prefix = "env:/development"
+aws_profile = "development-eu-west-2"
+kms_alias = "alias/development-eu-west-2/environment-services-kms"
+
+# service configs
+log_level = "debug"
+restricted_word_web_version = "1.0"
```
Comment (mwestacott) [inline terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:11]: Wondering about the `restricted_word_web_version` and `restricted_word_web_port` settings. Can't see those set in Mesos. Why are they here?

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+cookie_domain = "cidev.aws.chdev.org"
+cookie_name = "__SID"
+secure_cookie = "0"
+restricted_word_web_port = "3000"
```
[the remark this answers is not in the document]
Comment (Zain-Abbas1) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:15]: Some of these came from the service yaml file in docker-chs-development

**Comment by mattch1** [comment]
Not really sure what needs to go in here. I just copied over the fields from the variable.tf

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars (hunk 1 above)
Comment (mattch1) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars]: Only using these two variables.

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars (hunk 2 above)
Comment (mattch1) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars]: Removed

[the remark this answers is not in the document]
Comment (mattch1) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars]: Removed.

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars (hunk 3 above)
[the remark this answers is not in the document]
Comment (mattch1) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars]: Also removed

**Review by mwestacott — APPROVED** [review]

**Review by markpit — APPROVED** [review]

**Review by Zain-Abbas1 — APPROVED** [review]

File: terraform/groups/ecs-service/variables.tf [hunk 4]
```diff
@@ -51,10 +51,6 @@ variable "log_level" {
   type        = string
   description = "The log level for services to use: trace, debug, info or error"
 }
-variable "restricted_word_web_version" {
```
Comment (tobafolaranmi-ch) [inline terraform/groups/ecs-service/variables.tf:54]: Please add this back in. It is needed by the pipeline which passes the release version referenced in main.tf  to the ecs-service module

Comment (mattch1) [reply terraform/groups/ecs-service/variables.tf]: Added back, presumably we do not need the port variable.

**Review by tobafolaranmi-ch — APPROVED** [review]
LGTM

**Pull request merged**